servers/mcp-neo4j-cypher/README.md

```diff
@@ -164,6 +160,17 @@ source .venv/bin/activate  # On Unix/macOS
 uv pip install -e ".[dev]"
 ```
 
+3. Run Integration Tests
+
+**CLOSE ANY LOCAL NEO4J DATABASES BEFORE RUNNING TESTS**
```

Comment:
we should not require that, but instead use a safe way of running databases that doesn't conflict with existing db's

Reply:
too dangerous otherwise